tests/ember_debug/profile-manager-test.js

```diff
@@ -1,8 +1,246 @@
+      'component:test-bar',
+      EmberComponent.extend({
+        tagName: '',
+        toString() {
+          return 'App.TestBarComponent';
+        },
+      })
+    );
+
+    /*
+    Setting line-height to normal because normalize.css sets the
+    html line-height to 1.15. This seems to cause a measurement
+    error with getBoundingClientRect
+    */
+    this.owner.register(
+      'template:application',
+      hbs(
+        '<div class="application" style="line-height: normal;">{{outlet}}</div>',
+        { moduleName: 'my-app/templates/application.hbs' }
+      )
+    );
+    this.owner.register(
+      'template:simple',
+      hbs('Simple {{test-foo}} {{test-bar}}', {
+        moduleName: 'my-app/templates/simple.hbs',
+      })
+    );
+    this.owner.register(
+      'template:posts',
+      hbs('Posts', { moduleName: 'my-app/templates/posts.hbs' })
+    );
+    this.owner.register(
+      'template:components/test-foo',
+      hbs('test-foo', {
+        moduleName: 'my-app/templates/components/test-foo.hbs',
+      })
+    );
+    this.owner.register(
+      'template:components/test-bar',
+      hbs(
+        '<!-- before --><div class="another-component"><span>test</span> <span class="bar-inner">bar</span></div><!-- after -->',
+        { moduleName: 'my-app/templates/components/test-bar.hbs' }
+      )
+    );
+  });
+
+  test('ProfileManager can be created', function (assert) {
+    const profileManager = new ProfileManager();
+
+    assert.ok(!!profileManager, 'it was created');
+    assert.equal(profileManager.profiles.length, 0, 'it has no profiles');
+  });
+
+  test('Highlights for render are shown as expected', async function (assert) {
+    await visit('/posts');
+
+    // enable highlight
+    run(() =>
+      EmberDebug.port.trigger('render:updateShouldHighlightRender', {
+        shouldHighlightRender: true,
+      })
+    );
+
+    async function highlightsPromise() {
+      const observedHighlights = [];
+      const observer = new MutationObserver(function (records) {
+        records.forEach((record) => {
+          record.addedNodes.forEach((node) => {
+            if (node.className === 'ember-inspector-render-highlight') {
+              observedHighlights.push(node);
+            }
+          });
+        });
+      });
+      observer.observe(document.body, { childList: true });
+      await visit('/simple');
+      await waitUntil(() => observedHighlights.length === 2, { timeout: 2000 });
+      observer.disconnect();
+      return observedHighlights;
+    }
+
+    const newHighlights = await highlightsPromise();
+
+    const simpleComponent = find('.simple-component');
+    const anotherComponent = find('.another-component');
+
+    const [simpleHighlight, anotherHighlight] = newHighlights;
+
+    const simpleStyle = simpleHighlight.style;
+    const simpleExpected = simpleComponent.getBoundingClientRect();
+    assert.ok(simpleHighlight, 'simpleHighlight is visible');
+    assert.equal(
```

Comment:
let's have one unified asser functon for this two cases (likely we will add more cases to check)

```js

function assertNodeSizes(synthetic, real) {
    const style = synthetic.style;
    const box = real.getBoundingClientRect();
    const dimensions = [
        ['left', 'x'],
        ['top', 'y'],
        ['width', 'width'],
        ['height', 'height']
    ];
    for (const [styleKey, boxKey] of dimensions) {
        assert(
            getRounded(style[styleKey]),
            getRounded(box[boxKey]),
            `same ${boxKey} as component`
        )
    }
}
```